Cartfile

```diff
@@ -11,3 +11,4 @@ github "LoopKit/LogglyService" "dev"
 github "LoopKit/MKRingProgressView" "appex-safe"
 github "dexman/Minizip" "master"
 github "instacart/TrueTime.swift"
```

Comment:
Pete - These Cartfile values in this PR and my other PRs probably will need your attention before merge. I'm never quite sure how to manage the files during the development cycle as they become quite cumbersome to keep up-to-date with the current versions. 

Do we even rely on these Cartfile versions or maybe these should not be updated until just before you merge?

Reply:
And maybe I see my answer here. Looks like NightscoutService is way out of day. So I suppose we are not updating these.

Reply:
Correct. Carthage is being dropped in favor of Workspace builds.